### ISSUE TITLE
Implement a custom editor

### ISSUE THREAD

**Issue by FinalLody** [Calsign/APDE#16]
EditText is too slowly, a good solution is to write a Custom Editor extend View or SurfaceView such as AIDE.

**Comment by Calsign** [comment]
Yes, I should probably do this eventually. Unfortunately, there are more important things to work on right now.
